transaction.go

```diff
@@ -561,27 +561,28 @@ func (tx *Transaction) ProcessRequestBody() (*types.Interruption, error) {
 	}
 
 	rbp := tx.Variables.ReqbodyProcessor.String()
-
 	// Default variables.ReqbodyProcessor values
 	// XML and JSON must be forced with ctl:requestBodyProcessor=JSON
-	if tx.ForceRequestBodyVariable {
-		// We force URLENCODED if mime is x-www... or we have an empty RBP and ForceRequestBodyVariable
+	if rbp == "" {
+		if !tx.ForceRequestBodyVariable {
+			// so there is no bodyprocessor, we don't want to generate an error
+			tx.WAF.Rules.Eval(types.PhaseRequestBody, tx)
+			return tx.Interruption, nil
+		}
+
+		// We force URLENCODED if we have an empty RBP and ForceRequestBodyVariable
 		rbp = "URLENCODED"
 		tx.Variables.ReqbodyProcessor.Set(rbp)
 	}
 	tx.WAF.Logger.Debug("[%s] Attempting to process request body using %q", tx.ID, rbp)
-	rbp = strings.ToLower(rbp)
```

Comment:
rbp must be handled as lowercase

Reply:
https://github.com/corazawaf/coraza/pull/441/files#diff-63ab601287b8b9c040760fe0bdd288f55b73f37cd7e4f1e519bea2bd43a18bbaL579 already lowercase this.